components/channel_info_rhs/channel_info_rhs.tsx

```diff
@@ -83,7 +83,7 @@ const ChannelInfoRhs = ({
     actions,
 }: Props) => {
     const currentUserId = currentUser.id;
-    const channelURL = getSiteURL() + '/' + currentTeam.name + '/channels/' + channel.name;
+    const channelURL = getSiteURL() + '/' + currentTeam?.name + '/channels/' + channel.name;
```

Comment:
why is this change needed?

Reply:
when mattermost is coming back from the system config views the currentTeam is not loaded yeat, so it is undefined.

if i remove this react will break completely and not render anything because of that.

with this it will resolve itself in the second try reading from the url, at least that was as much as i could gather from debugging it.

Reply:
this is actually a bug, something we should be handling better. Thank for investigating this.

Reply:
this is now handled in the code, you may revert this

Reply:
done